Add close inactive task to crontab

tbrisker opened the pull request.

ehelms commented on puppet/modules/prprocessor/manifests/init.pp:
```diff
@@ -65,6 +64,14 @@
     require     => Package[$packages],
   }
 
+  cron { 'close inactive':
```
General question, cron is quite untrackable. Feels like whenever I see them we should be preferring something like a Jenkins job that can be monitored for failures. Thoughts?

tbrisker commented:
@ekohl addressed your comments; regarding monitoring - i agree this isn't ideal but hopefully this script specifically woulnd't need much monitoring, it was already running with cron on the old openshift infra.

ehelms commented:
@ekohl Harder because of SSH access requirement?

ekohl commented:
@ehelms yes. Because we have community slaves that means they have pretty much full access to our infra. The PR processor has some API tokens with a fair amount of access. While it's true that they already have access via other ways, we should be closing those ways down rather than opening up more ways.

ehelms commented:
I thought we:

 a) restricted ssh'ing to only be run on the master
 b) locked down what scripts can be run on the remote boxes via ssh for the jenkins user

ohadlevy commented on puppet/modules/prprocessor/manifests/init.pp:
```diff
@@ -65,6 +64,15 @@
   }
 
+  cron { 'close inactive':
+    command     => "cd ${app_root} && bundle exec ruby scripts/close_inactive.rb",
```
shouldn't it be bundle exec script/close_inactive.rb ?

ohadlevy commented on puppet/modules/prprocessor/manifests/init.pp:
```diff
@@ -65,6 +64,15 @@
+    environment => ["HOME=${app_root}"],
+    hour        => 1,
+    minute      => 23,
+    require     => Exec['install prprocessor']
```
dont you need the ENV=GITHUB_OAUTH_TOKEN too?

tbrisker replied:
I'm not sure, where would i get the token from?

ekohl replied:
You could do `GITHUB_OAUTH_TOKEN='${github_oauth_token}' bundle exec`. I think cron files per user can be considered secure since you can't read other users' crontabs.

ekohl commented:
@tbrisker could you have a look at the comments?

tbrisker commented:
Pushed an updated version that hopefully addresses all the comments. IIRC, there were some issues with the script itself that @ohadlevy was looking into, i'm not sure if they're fixed yet or not - if not we should wait to fix those first.
